### DISCUSSION TITLE
Enable Resizable bar in BIOS using modGRUBShell.

### DISCUSSION THREAD

**Discussion by msprestigepro** [xCuri0/ReBarUEFI#243] [Q&A]
i cant see enable resizable bar options in the bios after i added the dxe module, i have samsung b75s1 motherboard how do i enable like like it did with above 4g decoding var offset? using ifr extractor no data at all to extract. so i cant find the offset value.

**Comment by xCuri0** [comment]
you do it with rebarstate application, not in bios.
